python/cli/prelude_cli/views/iam.py

```diff
@@ -14,27 +14,42 @@ def iam(ctx):
     ctx.obj = IAMController(account=ctx.obj)
 
 
-@iam.command("create-account")
+@iam.command("account")
 @click.pass_obj
 @pretty_print
-@click.confirmation_option(
-    prompt="Overwrite local account credentials for selected profile?"
-)
-def register_account(controller):
-    """Register a new account"""
-    email = click.prompt("Enter your email")
-    company = click.prompt("Enter your associated company")
-    name = click.prompt("(Optional) Enter your name", default="", show_default=False)
-    slug = click.prompt(
-        "(Optional) Enter an unique human-readable identifier for your account",
-        default="",
-        show_default=False,
-    )
-    with Spinner(description="Creating new account"):
-        data = controller.new_account(
-            user_email=email, company=company, user_name=name, slug=slug
-        )
-    return data, "Check your email to verify your account"
+def describe_account(controller):
+    """Get account details"""
+    with Spinner(description="Fetching account details"):
+        return controller.get_account()
+
+
+@iam.command("list-accounts")
+@click.pass_obj
+@pretty_print
+def describe_account(controller):
```

Comment:
name.
also, think we should do a separate module for the User-based cmds (list-accounts, update-user, purge-user) vs the account-based ones?